OpenDreamRuntime/Procs/DebugAdapter/DreamDebugManager.cs

```diff
@@ -12,12 +14,57 @@ sealed class DreamDebugManager : IDreamDebugManager {
     [Dependency] private readonly IProcScheduler _procScheduler = default!;
     [Dependency] private readonly IBaseServer _server = default!;
 
+    // Setup
     private DebugAdapter? _adapter;
-    private readonly Dictionary<string, List<Breakpoint>> _breakpoints = new();
+    private string RootPath => _resourceManager.RootPath ?? throw new Exception("No RootPath yet!");
+    private bool _stopOnEntry = false;
+
+    // State
+    private bool Stopped = false;
```

Comment:
This should be renamed to `_stopped` to match the naming convention of private fields.